docs/DeveloperGuide.md

```diff
@@ -919,38 +919,253 @@ testers are expected to do more *exploratory* testing.
+       Expected: The address of the first stall and its associated reviews edited. Details of the edited stall shown in the status message. GUI updates to show the edited details.
+   
+    4. Test case: `sedit 1 n/Alice's Chicken Rice`<br>
+       Expected: No stall is edited. Error details shown in the status message. GUI does not update.
+
+    5. Other incorrect stall edit commands to try: `sedit`, `sedit 1`, `sedit x n/Alice Chicken Rice`, `sedit x a/Chicken Street 123`(where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Editing a review
+
+1. Editing a review while all reviews are being shown
+
+    1. Prerequisites: List all reviews using the `rlist` command. Multiple reviews in the list.
+
+    2. Test case: `redit 1 d/20/09/2022`<br>
+       Condition: There must not exist a review on the same stall with the same details (date, content and tags) as the edited review's details<br>
+       Expected: The date of the first review edited. Details of the edited review shown in the status message. GUI updates, the edited review is moved to the back of the list with the edited details shown.
+
+    3. Test case: `redit 1 c/The chicken was dry`<br>
+       Condition: There must not exist a review on the same stall with the same details (date, content and tags) as the edited review's details<br>
+       Expected: The content of the first review edited. Details of the edited review shown in the status message. GUI updates, the edited review is moved to the back of the list with the edited details shown.
+   
+    4. Test case: `redit 1 d/2022/09/20`<br>
+       Expected: No review is edited. Error details shown in the status message. GUI does not update.
+
+    5. Other incorrect review edit commands to try: `redit`, `redit 1`, `redit x d/20/09/2022`, `redit x c/The chicken was dry`(where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Find stalls
+
+1. Finding stalls by name and/or tags
+
+    1. Prerequisites: At least one stall entry exists that we are finding with name and/or tag exists.
+   
+    2. Test case: `sfind n/chicken`<br>
+       Condition: There is a stall with the word `chicken` in its name<br>
+       Expected: Status message responds with the number of stalls that matches the name in command. GUI updates to show stalls that matches the name in command. 
+   
+    3. Test case: `sfind t/opendaily`<br>
+       Condition: There is a stall with the tag `opendaily`<br>
+       Expected: Status message responds with the number of stalls that matches the tag in command. GUI updates to show stalls that matches the tag in command.
+   
+    4. Test case: `sfind n/chicken t/opendaily`<br>
+       Condition: There is a stall with the word `chicken` in its name or with the tag `opendaily`<br>
+       Expected: Status message responds with the number of stalls that matches the name and tag in command. GUI updates to show stalls that matches the name or tag in command.
```

Comment:
"Expected: Status message responds with the number of stalls with `chicken` as a word in the name and/or has `opendaily` as a tag. GUI updates to show these stalls."

Note there were two instances of "name and tag" on this line. Likewise, line 1098.

One other thing I think can be improved is that "matches the name in the command" so here is a recommended wording that I'll suggest.